Ejer3LinqPractice/Program.cs

```diff
@@ -1,15 +1,149 @@
 ﻿using System;
+using System.IO;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
 
 namespace Ejer3LinqPractice
 {
     class Program
     {
         static void Main(string[] args)
         {
+            String archivo = File.ReadAllText("Heroes.json");
+            List<Heroe> heroes= JsonConvert.DeserializeObject<List<Heroe>>(archivo);
+
+            /*Console.WriteLine(Ejer2(heroes));
+            Ejer3(heroes);
+            Ejer4(heroes);
+            Ejer5(heroes);
+            Ejer6(heroes);
+            Ejer7(heroes);
+            Ejer8(heroes);
+            Ejer9(heroes);
+            Ejer10(heroes);
+            Ejer11(heroes);*/
+            Ejer12(heroes);
+            Console.ReadKey();
+        }
+        public static String Ejer2(List<Heroe> heroes) {
+            
+            var femenino=heroes.Count(x => x.Gender == "Female");
+            var masculino=heroes.Count(x => x.Gender == "Male");
+            String cadena = "Heroes masculinos:" + masculino+" Heroes femeninos:"+femenino;
+            return cadena;
+        }
+
+        public static void Ejer3(List<Heroe> heroes)
+        {
+            var ej3 = heroes.Where(x => x.Level >7);
+            Console.WriteLine("Los 10 primeros heroes con nivel superior a 7 son:");
+            int contador = 0;
+            foreach (var a in ej3)
```

Comment:
@DaniRomeroR En caso de que vayas a hacer un contador, usa For en vez de Foreach

Reply:
@DaniRomeroR Además que es muy ineficiente. Muestras los 10 primeros pero recorre toda la colección